### PR TITLE
Add structures to old checkpoints

### PR DESCRIPTION
**Pull request by Nash2325138**
## Why do we need this PR?
* The old checkpoint has no structures like "generator.course_net.downsample_module.conv1" but just "conv1". We need to add those prefixes into the `state_dict`'s keys of that ckpt.

* Also, in the config of that checkpoint there is a deprecated loss `TemporalWarpingLoss` which needs `flownet_checkpoint_path` as a positional argument. We need to provide a default value to load that checkpoint.

## Others
* I have added the checkpoint (transformed by `scripts/FVI_checkpoint_compatibility_transform.py`) that we used to report scores on the ICCV2019 paper in the [google drive](https://drive.google.com/open?id=1uva9yI8yYKivqi4pWcyZLcCdIt1k-LRY) we put in the readme.

### CODE REVIEW

**Comment by amjltc295** [comment]
@Nash2325138 could you fix #8 and #22? The naming is a bit confusing.

**Comment by Nash2325138** [comment]
> @Nash2325138 could you fix #8 and #22? The naming is a bit confusing.

Fixed
